packages/property-grid/src/PropertyGridManager.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+*--------------------------------------------------------------------------------------------*/
+/** @module Common */
+
+import * as i18next from "i18next";
+import { I18N } from "@bentley/imodeljs-i18n";
+import { UiError, getClassName } from "@bentley/ui-abstract";
+import { FindSimilar } from "@bentley/itwin-find-similar/lib/api/FindSimilar";
```

Comment:
FindSimilar doesnt seem to be used here. But it would also need to be initialized prior to being used in this pkg. Maybe do a lazy init for it in here?